api/v1alpha1/httpsedge_types.go

```diff
@@ -68,6 +68,9 @@ type HTTPSEdgeSpec struct {
 
 	// Routes is a list of routes served by this edge
 	Routes []HTTPSEdgeRouteSpec `json:"routes,omitempty"`
+
```

Comment:
I also see this terminateAt in the api client https://github.com/ngrok/ngrok-api-go/blob/da8b5e25f3a1272c3106bfb06b05fd3847fa201c/datatypes.go#L1558 and its in the api response body https://ngrok.com/docs/api/resources/https-edge-tls-termination-module#fields but not the update body. Any idea what's going on there?

Reply:
I believe it is just a common config type that is returned for TLSEdges and HTTPSEdges. In the case of a HTTPSEdge, it will be terminated at edge. In the case of TLSEdge, it will be terminated at the upstream. You are correct in that the update body does not allow for you setting `terminateAt`. I noticed that as well while working on this.

Reply:
ahh i didn't see it was shared. that makes sense that the behaviour would be different between the 2 of those and the user isn't in control